protos/action_server/action_server.proto

```diff
@@ -55,6 +59,14 @@ message SetAllowableFlightModesRequest
     AllowableFlightModes flight_modes = 1;
 }
 
+message SetArmedStateRequest {
+    bool is_armed = 1; // Is armed now?
+}
+
+message SetFlightModeRequest {
+    FlightMode flight_mode = 1;
```

Comment:
Probably needs a docstring.

Reply:
Agreed, makes sense. I'll add it right now.

Reply:
Done